docs/concepts.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# scipp-widgets concepts\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inputs\n",
+    "The input class describes how an input widget should be constructed and how its input should be processed. There are a number of generic classes provided. If you wish to make your own however the interface that needs to be impliment is one property and two methods.These are:\n",
+    " * The `name` property, this should return a string and corrospond to the name of the underyling function input this input corrosponds to.\n",
+    " * The `validator` method. This takes a single input and returns a single input. This method is called on the input value as a pre-processing step before it it passed to the underlying function. This function should do any neccecary pre-processing and throw a ValueError if the validation fails. See the validator section below for more details.\n",
```

Comment:
> This takes a single input and returns a single input.
This does not make much sense to me...

Reply:
I'll re-word. I think `This takes a single input and returns a single output` is clearer and closer to the actual meaning.